### DISCUSSION TITLE
Question on P2P network setup

### DISCUSSION THREAD

**Discussion by telecomsxchange** [ambianic/peerfetch#16] [Q&A]
@ivelin I have a question on P2P network setup step. 

In your example your connecting to signaling service hosted on heroku to register the peer. Which piece of the software is responsible for the signaling service "registrar"? This part I'm a bit confused about.

**Comment by ivelin** [comment]
Great question. Here is the repo for the signaling server:
https://github.com/ambianic/ambianic-pnp

**Reply by telecomsxchange** [reply]
Thanks @ivelin all clear.
